homeassistant/components/thethingsnetwork/const.py

```diff
@@ -0,0 +1,15 @@
+"""The Things Network's integration constants."""
+
+from homeassistant.const import Platform
+
+DOMAIN = "thethingsnetwork"
+TTN_API_HOSTNAME = "eu1.cloud.thethings.network"
+
+PLATFORMS = [Platform.SENSOR]
+
+CONF_HOSTNAME = "hostname"
+CONF_API_KEY = "access_key"
+CONF_APP_ID = "app_id"
+
+POLLING_PERIOD_S = 60
+FIRST_FETCH_H = 24
```

Comment:
This isn't used now right

Reply:
POLLING_PERIOD_S is still used in coordinator as the TTN client does not yet implement pushing updates in all cases.

Reply:
But the other one

Reply:
Yes, this other one I removed it and left the client default to be used.